### PR TITLE
Add initial skeleton for the aos_core application

### PR DESCRIPTION
**Pull request by oleksiimoisieiev**
Signed-off-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (al1img) [inline CMakeLists.txt:1]: Use following header for cmake files:
```
#
# Copyright (C) 2023 Renesas Electronics Corporation.
# Copyright (C) 2023 EPAM Systems, Inc.
#
# SPDX-License-Identifier: Apache-2.0
#
```

File: src/main.c [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/*
```
Comment (al1img) [inline src/main.c:1]: Use the following header for c files:
```c
/*
 * Copyright (C) 2023 Renesas Electronics Corporation.
 * Copyright (C) 2023 EPAM Systems, Inc.
 *
 * SPDX-License-Identifier: Apache-2.0
 */
```

File: src/main.c [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stddef.h>
```
Comment (al1img) [inline src/main.c:6]: Empty line after license header.

File: src/main.c [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+#include <stdlib.h>
```
Comment (al1img) [inline src/main.c:7]: Remove unused includes.

File: west.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (al1img) [inline west.yml:1]: Update license.

File: CMakeLists.txt (hunk 1 above)
Comment (oleksiimoisieiev) [reply CMakeLists.txt:1]: +

File: src/main.c (hunk 3 above)
Comment (oleksiimoisieiev) [reply src/main.c:6]: +

File: src/main.c (hunk 4 above)
Comment (oleksiimoisieiev) [reply src/main.c:7]: +

File: west.yml (hunk 5 above)
Comment (oleksiimoisieiev) [reply west.yml:1]: +

File: west.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+  remotes:
+    - name: xen-troops
+      url-base: https://github.com/xen-troops/
+
+  projects:
+    - name: zephyr
+      remote: xen-troops
+      revision: aos-wip
+      west-commands: scripts/west-commands.yml
```
[the remark this answers is not in the document]
Comment (oleksiimoisieiev) [reply west.yml:22]: +

**Review by al1img — APPROVED** [review]

**Pull request merged**